articles/openstack/ostack-how-use-octavia.md

```diff
@@ -0,0 +1,225 @@
+---
+title: How to use Octavia LBaaS on UKCloud for OpenStack
+description: Provides information on using Octavia Load Balancing as a Service (LBaaS) within your OpenStack environment
+services: openstack
+author: Blane Bramble
+reviewer: 
+lastreviewed: 
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Use Octavia Load Balancing as a Service
+toc_fullpath: How To/ostack-how-use-octavia.md
+toc_mdlink: ostack-how-use-octavia.md
+---
+# Using Octavia Load Balancing as a Service (LBaaS)
+> [!NOTE]
+> TBD: region restrictions
```

Comment:
Need to provide the region restrictions (or remove this note if there are no restrictions)

Reply:
Updated to specify OSP10 and earlier not supported